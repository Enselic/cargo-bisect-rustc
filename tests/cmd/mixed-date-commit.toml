bin.name = "cargo-bisect-rustc"
args = "--start 2024-03-01 --end 5257aee7dd163d21d32fa904578d4fb0f4c91b79"
status = "failed"
