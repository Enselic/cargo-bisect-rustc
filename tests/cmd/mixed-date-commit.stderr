ERROR: cannot take different types of bounds for start/end, got start: Some(Date(2024-03-01)) and end Some(Commit("5257aee7dd163d21d32fa904578d4fb0f4c91b79"))
